src/prometheus/definition.go

```diff
@@ -56,7 +56,7 @@ func defaultEntityTypeFromLabelValue( //nolint: cyclop
 	key string, group, groupLabel string, rawEntityID string, g definition.RawGroups, clusterName string,
 ) (string, error) {
 	switch groupLabel {
-	case "namespace", "node": //nolint: goconst
+	case "namespace", "node", "persistentvolume": //nolint: goconst
```

Comment:
This is the function that creates the entity name. For example, for `CronJob`, you could have `entityName` = `k8s:dev-cluster-htroisi-2:nri-k8s-dev:cronjob:e2e-resources-cronjob`. We have to add an exception for `PersistentVolume` because PV doesn't have an associated namespace name.

Reply:
Makes sense, thanks for the explanation